apps/docs-app/docs/getting-started.md

```diff
@@ -165,3 +165,33 @@ The server for the API/SSR build artifacts is located in the `dist/apps/analog-a
 
 </TabItem>
 </Tabs>
+
+## Upgrade your existing application
```

Comment:
This should be a separate page under the `Features` group.

You can create a folder for `updating` [in here](https://github.com/analogjs/analog/tree/main/apps/docs-app/docs/features) and an `overview.md` for the content.

The config file for the sidemenu is here https://github.com/analogjs/analog/blob/main/apps/docs-app/sidebars.js

Reply:
ok, i will fix that. Thanks